.github/workflows/check.yml

```diff
@@ -53,7 +69,90 @@ jobs:
         run: |
           cargo test
           
-      
-      - name: Check if Rustdocs can be Generated
+
+      - name: Check if Rustdocs Builds
         run: |
           cargo doc
+
+
+  #
+  # Build documentation website
```

Comment:
what is all this commented stuff here?

Reply:
I was trying to upload the rustdoc on NFT.Storage, but it wasn't uploading successfully. So I commented it out

Should I remove it?